source/features/mark-merge-commits-in-list.tsx

```diff
@@ -45,34 +47,36 @@ export function getCommitHash(commit: HTMLElement): string {
 	return getCommitLink(commit)!.pathname.split('/').pop()!;
 }
 
-async function init(): Promise<void> {
-	const isPRConversation = pageDetect.isPRConversation();
-	const pageCommits = $$([
-		'.listviewitem',
-
-		// Old view style (before November 2023)
-		'.js-commits-list-item', // `isCommitList`
-		'.js-timeline-item .TimelineItem:has(.octicon-git-commit)', // `isPRConversation`, "js-timeline-item" to exclude "isCommitList"
-	]);
+const isPRConversation = pageDetect.isPRConversation();
```

Comment:
This can't be here because the URL will change and this will not be executed again.

It's better to inline `updateCommitIcon` again